### ISSUE TITLE
Types & Grammar (Tipos e Gramática) - Capítulo 2

### ISSUE THREAD

**Issue by cez-aug** [cez-aug/You-Dont-Know-JS#10]

**Comment by JoaoGFarias** [comment]
Quero traduzir esse capítulo

**Comment by cezaraugusto** [comment]
Legal @JoaoGFarias, tá contigo.

Caso tenha dúvidas sobre a tradução de certos termos, veja o nosso [WORDREFERENCE](https://github.com/cezaraugusto/You-Dont-Know-JS/blob/portuguese-translation/WORDREFERENCE.md). Peço que consulte também o #68 para ver como estamos fazendo as traduções. 👍

**Comment by opchaves** [comment]
Olá!
Gostaria de continuar a tradução deste capítulo.

**Comment by cezaraugusto** [comment]
Legal @paulochavesbr, tá contigo.

Caso tenha dúvidas sobre a tradução de certos termos, veja o nosso [WORDREFERENCE](https://github.com/cezaraugusto/You-Dont-Know-JS/blob/portuguese-translation/WORDREFERENCE.md). Peço que consulte também o #68 para ver como estamos fazendo as traduções. 👍

**Comment by dulcetti** [comment]
Oi @cezaraugusto, vi alguns erros gramaticais, como faço para corrigir? Tem alguém pegando isso ou poss fazer um fork e Pull Request?

**Comment by cezaraugusto** [comment]
@dulcetti fique à vontade. O @paulochavesbr tá nessa continuação, mas pode mandar um PR com correções do que já foi feito e não está bom. Obrigado!

**Comment by gabsprates** [comment]
Tem alguém com a tradução deste capítulo? Se não, seria interessante colocar com a label "Disponível para tradução", não?

**Comment by dulcetti** [comment]
@gabsprates eu acho que poderia ter uma label quando estiver sendo traduzido, porque deve ser em menos quantidade, com isso, menos trabalhoso :D

**Comment by gabsprates** [comment]
@dulcetti você fala, além da `Em Progresso`? Ter uma simbolizando que além da tradução em progresso, alguém está traduzindo alguma parte, isso?

**Comment by dulcetti** [comment]
@gabsprates verdade, agora que eu vi que tem o `Em progresso`. Mas acho que poderia ser acompanhado de perto, ficar mais esclarecido como funcionam as coisas. Não sei se tem alguém no momento mexendo nesse capítulo, por exemplo.

**Comment by kelvynrisso** [comment]
Olá,
Posso continuar a tradução deste capítulo?

**Comment by cezaraugusto** [comment]
@kelvynrisso desculpa a demora. Se tiver interesse ainda da um alo que eu mudo a etiqueta para "em progresso".

**Comment by kelvynrisso** [comment]
@cezaraugusto, tenho interesse sim. Irei iniciar a tradução então. Obrigado.

**Comment by alinedmelo** [comment]
Vou ficar com esse ;D

**Comment by cezaraugusto** [comment]
alterando status para `Disponível para tradução` por conta do tempo. se eu estiver enganado, me dá um alô!

**Comment by osergioneto** [comment]
Olá gente, gostaria de traduzir esse capitulo :)

**Comment by cezaraugusto** [comment]
oi @osergioneto desculpe a demora.

pode dar um alô de novo caso ainda tenha interesse? daí eu mudo o _status_ da _issue_. obrigado!

**Comment by caioreis123** [comment]
posso pegar esse capítulo @cezaraugusto ?

**Comment by cezaraugusto** [comment]
oi @caioreis123, pode sim. vou alterar a etiqueta para "em progresso"

**Comment by lucas-wa** [comment]
Olá! Posso terminar de traduzir? Posso fazer isso ao mesmo tempo em que leio.

**Comment by lucas-wa** [comment]
Olá! O capítulo ainda está sendo traduzido? A etiqueta diz que sim, mas a partir de "Special Equality" os textos estão em inglês. Posso traduzir enquanto termino de ler.

**Comment by dulcetti** [comment]
Por mim pode mandar brasa, só ver com o @cezaraugusto, mas acredito que nem esteja sendo traduzido, já que desde 2019 ninguém interage nessa parte aqui ;)

**Comment by lucas-wa** [comment]
Certo, vou aguardar a resposta
